x-pack/plugins/observability_solution/entity_manager/server/lib/entities/built_in/services.ts

```diff
@@ -104,17 +104,12 @@ export const builtInServicesFromLogsEntityDefinition: EntityDefinition =
       },
       {
         name: 'logErrorRate',
-        equation: 'A / B',
+        equation: 'A',
```

Comment:
We should also bump the definition version so the changes gets picked up by the upgrade logic

Reply:
You're refer to this version https://github.com/elastic/kibana/blob/8e3be0a325fcf95dfdba59463fef48b9d97dcc6c/x-pack/plugins/observability_solution/entity_manager/server/lib/entities/built_in/services.ts#L23 ? 

Do you have a changelog I should also update?

Reply:
> Do you refer to this version ?

yes

> Do you have a changelog I should also update?

no changelog atm, I thought I logged an issue to implement it but looks like I didn't. Let's ignore this for now I'll create a follow up

Reply:
https://github.com/elastic/kibana/pull/191959/commits/83fa7e16943f3bb9c01f758906101d8ea4e307e5